xbmc/guilib/FFmpegImage.cpp

```diff
@@ -487,23 +487,9 @@ bool CFFmpegImage::DecodeFrame(AVFrame* frame, unsigned int width, unsigned int
   AVColorRange range = frame->color_range;
   AVPixelFormat pixFormat = ConvertFormats(frame);
 
-  // assumption quadratic maximums e.g. 2048x2048
-  float ratio = m_width / (float)m_height;
-  unsigned int nHeight = m_originalHeight;
-  unsigned int nWidth = m_originalWidth;
-  if (nHeight > height)
-  {
-    nHeight = height;
-    nWidth = (unsigned int)(nHeight * ratio + 0.5f);
-  }
-  if (nWidth > width)
-  {
-    nWidth = width;
-    nHeight = (unsigned int)(nWidth / ratio + 0.5f);
-  }
-
-  struct SwsContext* context = sws_getContext(m_originalWidth, m_originalHeight, pixFormat,
-    nWidth, nHeight, AV_PIX_FMT_RGB32, SWS_BICUBIC, NULL, NULL, NULL);
+  struct SwsContext* context =
```

Comment:
```suggestion
  SwsContext* context =
```

Reply:
Done.

Reply:
Nope, the `struct` is still there :wink:

Reply:
Ah, my bad. Fixed.